### PR TITLE
Bug(DS-2132): FxA accounts dashboard is broken client id not found

### PR DESCRIPTION
**Pull request by kik-kik**
# bug(DS-2132): FxA accounts dashboard is broken client id not found

Seems the change resulted in expected changes:
https://github.com/mozilla/looker-hub/blob/kik-generation-test-1/firefox_accounts/views/growth_accounting.view.lkml#L103-L108

### CODE REVIEW

File: custom-namespaces.yaml [hunk 1]
```diff
@@ -334,6 +334,7 @@ firefox_accounts:
           table: mozdata.firefox_accounts.events_daily
     growth_accounting:
       type: growth_accounting_view
+      primary_key_field: user_id
```
Comment (fbertsch) [inline custom-namespaces.yaml:337]: nit: this field isn't a primary key, it has to be combined with the `submission_date` to qualify as that. Maybe call it `identifier_field`?

Comment (kik-kik) [reply custom-namespaces.yaml]: fair enough, renamed the field.

**Review by scholtzan — APPROVED** [review]
looks good to me

**Comment by kik-kik** [comment]
Finally managed to generate the dev branch and check it in Looker. Seems this fixes the issue identified in the ticket.

**Pull request merged**